### DISCUSSION TITLE
Twitter spaces with folks from subsquid on April 25-30 ( exact date TBA )

### DISCUSSION THREAD

**Discussion by damskyftw** [kodadot/nft-gallery#2793] [General]
## Twitter spaces 25-30 april (date TBA depending on attendants)

Hey, as i had nice call today with folks from subsquid, i'd actually like to encourage our developooors who are maintaining KodaDot to join us on this session. Since subsquid is more of a developerish project, i'd like to drive this discussion this way.

Maybe @yangwao or @vikiival could join, but i guess also other contributors could make some time and share their knowledge and experiences from the space. I believe it will be pretty nice fireside chat since subsquid also has vibrant community.

## Topic

That's why i actually need your help! I believe sharing your experience in the web3 developer road will be pretty helpful but other than that, i'd like you to come up with some topics that could be interesting for both communitites, can be surely developer focused, nothing wrong with that. I'm also sending this discussion over to folks from subsquid, let's think on some topics and take care of the details here :)

### Looking forward!
![I'm coding_KODA_APNG](https://user-images.githubusercontent.com/92786519/162221445-a79305b2-b48f-40be-8cd9-6f2c38b238da.png)

**Comment by vikiival** [comment]
Someday like 28, 29 would really wokr for me :)

**Comment by DLangellotti** [comment]
I think it'd be great to do something about recruiting developers for an open-source project. A bit meta, I know! But a great topic that brings communities together well.

**Reply by yangwao** [reply]
Sounds good to me! We are actually trying to formalize local guilds to help around so definitely a great topic to chat about! As we use bounties in KodaDot tho :)

**Reply by DLangellotti** [reply]
We're just starting with bounties now, talked a bit about what we're up to now with Damsky the other day. We're thinking about doing it a bit differently depending on the type of tasks. We want to do a Github-based system like what you have for building 'internally,' while we've got an MVP developed for an interface where devs can reserve tasks to build Squids or analytics UIs, and eventually to deploy Archives.
